src/cards.rs

```diff
@@ -845,6 +845,7 @@ impl GameState {
                 south,
                 west,
                 rules,
+                created_at: _,
```

Comment:
this line can be `..` instead to ignore the created_at field (and all other fields).